exchange/exchange_test.go

```diff
@@ -4809,10 +4854,12 @@ func TestMakeBidWithValidation(t *testing.T) {
 	for _, test := range testCases {
 		e.bidValidationEnforcement = test.givenValidations
 		sampleBids := test.givenBids
-		resultingBids, resultingErrs := e.makeBid(sampleBids, sampleAuction, true, ImpExtInfoMap, bidExtResponse, "", "")
+		nonBids := &nonBids{}
+		resultingBids, resultingErrs := e.makeBid(sampleBids, sampleAuction, true, ImpExtInfoMap, bidExtResponse, test.givenSeat, "", nonBids)
 
 		assert.Equal(t, 0, len(resultingErrs), "%s. Test should not return errors \n", test.description)
 		assert.Equal(t, test.expectedNumOfBids, len(resultingBids), "%s. Test returns more valid bids than expected\n", test.description)
+		assert.Equal(t, test.expectedNonBids, nonBids, "%s. Test returns incorrect nonBids\n", test.description)
 	}
 }
```

Comment:
Nitpick: can you please wrap the tests execution in `t.Run` function?
Like this:
```go
//Run tests
	for _, test := range testCases {
		t.Run(test.description, func(t *testing.T) {
			e.bidValidationEnforcement = test.givenValidations
			sampleBids := test.givenBids
			nonBids := &nonBids{}
			resultingBids, resultingErrs := e.makeBid(sampleBids, sampleAuction, true, ImpExtInfoMap, bidExtResponse, test.givenSeat, "", nonBids)

			assert.Equal(t, 0, len(resultingErrs), "%s. Test should not return errors \n", test.description)
			assert.Equal(t, test.expectedNumOfBids, len(resultingBids), "%s. Test returns more valid bids than expected\n", test.description)
			assert.Equal(t, test.expectedNonBids, nonBids, "%s. Test returns incorrect nonBids\n", test.description)
		})
	}
```
This will follow genaral test approach and also allow per-test-case execution.

Reply:
Done